lib/http/preprocessors.js

```diff
@@ -67,7 +68,7 @@ const authHandler = ({ Sessions, Users, Auth, bcrypt }, context) => {
     return authBySessionToken(authHeader.substring(7), () => reject(Problem.user.authenticationFailed()));
 
   // Basic Auth, which is allowed over HTTPS only:
```

Comment:
can you please update this comment as well

Reply:
What would you recommend?

Reply:
How about "Basic Auth, which is allowed only over HTTPS and only if SSO is not enabled"? Or "Basic Auth, which is allowed only if certain conditions are true"? I think right now, the comment seems a little outdated since it mentions HTTPS but not SSO.

Reply:
PR at https://github.com/getodk/central-backend/pull/982